.circleci/config.yml

```diff
@@ -171,16 +272,28 @@ jobs:
           key: external-{{ arch }}-{{ .Branch }}-{{ epoch }}
           paths:
             - external
-
       - run: ccache --show-stats
+      - run:
+          name: rename artifacts
+          command: |
+            cp build/iroha-*.tar.gz build/iroha-darwin.tar.gz
```

Comment:
Maybe use specific filename here instead of wildcard?

Reply:
name contains `version`, and we use git commit hash as a version, so it is easier to just use asterisk.
There are no name collisions, since it is built on CI, which creates clean machine for every build

Reply:
This intent might be more clear for the reader with specifying variable explicitly.

Reply:
we don't use IROHA_VERSION yet, because we do not have versioning policy. Now dummy commit hash is used. 

If we implement versioning policy, now we do not need to change circle files. We just specify env var in circle ci.